docs/pages/installation/_index.md

```diff
@@ -12,12 +12,12 @@ Before installing Liqo, you should:
 * Provision the clusters you would like to use with Liqo.
 * Have a look to the [pre-install section](./pre-install), that presents some common patterns used to expose and interconnect clusters when using Liqo.
 
-#### Liqoctl 
+#### liqoctl 
 
 Liqoctl is the swiss-knife CLI tool to install and manager Liqo. 
 It is strongly recommended installing Liqo using Liqoctl since it provides customizations for supported providers.
```

Comment:
```suggestion
We strongly recommend to install Liqo using Liqoctl because it automatically handles the required customizations for each supported providers (e.g., AWS, EKS, etc.).
```